content/project-organization/quality-council/processes/issue-management.md

```diff
@@ -10,107 +10,122 @@ This document details the processes in place for the position of "Issues Manager
-- **Has all the needed information been provided?** The issue must be understandable and contain the required information as provided in the issue template, complemented with any other detail needed for reproduction.
-- **Can it be reproduced?** An issue can't be fixed if it can't be reproduced.
+- **Is the issue valid?** The issue must be either a bug report or a request for feature. We only cover the PrestaShop project (Core + native modules).
+- **Has all the needed information been provided?** The issue must be understandable and contain the required information as provided in the issue template, complemented with any other detail needed for reproduction.
+- **Can it be reproduced?** An issue can't be fixed if we can't reproduce it.
 
 Here are some tips:
 
-- **Don't provide free support.** Your job isn't to help the reporter fix their problem, but to determine if the problem is caused by a defect in the software (or in documentation).
-- **Be pragmatic about your efforts.** It's the reporter's responsibility to provide the information needed to reproduce the problem in an understandable way. Don't waste your energy doing guesswork or waiting for a response.
-- **Don't be a robot either.** Don't take these guidelines to the letter. If the reporter makes an effort, try to be flexible.
+- **Don't provide free support.** Your job isn't to help the reporter fix their problem, but to determine if the problem is caused by a defect in the software (or in documentation).
+- **Be pragmatic about your efforts.** It's the reporter's responsibility to provide the information needed to reproduce the problem in an understandable way. Don't waste your energy doing guesswork or waiting for a response.
+- **Don't be a robot either.** Don't take these guidelines to the letter. If the reporter makes an effort, try to be flexible.
 
 ## Issue state & labels
 
 GitHub has only two states for issues: open and closed. We use labels to keep track of issue type, intermediate state, severity and resolution.
 
-#### Main Labels
+### Main Labels
 
 1. **Type**
-   - **`Bug`** – a defect in the software
-   - **`Feature`** – a request for a new feature, a change in an existing functionality
-   - **`Task`** – an issue to track an action that’s not a fix or improve the software like to release a module or network infrastructure change. It’s a task mainly to be done by a maintainer dev.
-   - **`Epic`** - an epic is a group of issues around a bigger topic.
-
-
+    - **`Bug`** – a defect in the software
+    - **`Deprecated - Feature`** – a new feature, a change in an existing functionality (deprecated since April 2023)*
+    - **`Task`** – an issue to track an action that’s not a fix or improve the software like to release a module or network infrastructure changement. It’s a task mainly to be done by the maintainer dev.
+    - **`Epic`** – a epic is a group of issue that are going to
```

Comment:
Isn't there a stuff missing at the end of the sentence here?

Reply:
It's the same in Notion.